testing/test_integration_twisted.py

```diff
@@ -1,8 +1,10 @@
 import logging
-from ldclient.twisted import TwistedLDClient, TwistedConfig
+from ldclient.client import LDClient
+from ldclient.twisted_impls import TwistedConfig
```

Comment:
I also liked the aesthetics of importing `ldclient.twisted` instead of `ldclient.twisted_impls`